app/src/main/java/com/yagni/model/Create.java

```diff
@@ -3,30 +3,44 @@
 import java.sql.*;
 import com.yagni.controller.*;
 
+/*
+* This class Creates a new Item in the inventory
+*/
 public class Create {
-    
-    private static String productId, supplierId, quantity, salePrice, wholeSale;
 
-    public Create(String productIdIn, String quantityIn, String wholeSaleIn, String salePriceIn, String supplierIdIn){
+    // String Variables
+    private static String productId;
+    private static String supplierId;
+    private static String quantity;
+    private static String salePrice;
+    private static String wholeSale;
+
+    // Constructor for the Create class
+    // Takes in all field inputs
+    public Create(String productIdIn, String quantityIn, String wholeSaleIn, String salePriceIn, String supplierIdIn) {
         productId = productIdIn;
         supplierId = supplierIdIn;
         quantity = quantityIn;
         salePrice = salePriceIn;
         wholeSale = wholeSaleIn;
     }
 
+    // Method runs a sql insert statement
+    // Takes a DbConnection object as an argument
     public void addRecord(DbConnection linkDB) {
 
         // Inserting into the database
-        // Try catches SQLExecption
+        // Try catches SQLException
         try {
             // Creates a statement object
             Statement statement = linkDB.getConnection().createStatement();
             // Calling the execute method to execute an INSERT statement
-            statement.execute("INSERT INTO `product`(product_id, quanity, Whole_sale, Sale_cost, vendor_id) VALUES ('"+productId+"',"+quantity+","+wholeSale+","+salePrice+",'"+supplierId+"')");
+            statement.execute("INSERT INTO `product`(product_id, quanity, Whole_sale, Sale_cost, vendor_id) "
+                            + " VALUES ('"+productId+"',"+quantity+","+wholeSale+","+salePrice+",'"+supplierId+"')");
```

Comment:
nice line-wrap. Looks like you hit tab 4 times on it though. It should be 2 tabs past line 38. Alyssa has an example of how it should look on Line 362&363 under ManageInvGUI.java

Reply:
> nice line-wrap. Looks like you hit tab 4 times on it though. It should be 2 tabs past line 38. Alyssa has an example of how it should look on Line 362&363 under ManageInvGUI.java

I seen what you did here and I would be fine w/o the changes.

Reply:
Yeah I wanted line it up so it looked more like an SQL statement for easier readability.